BackofficeBundle/Resources/public/coffee/setUpCallAjax.coffee

```diff
@@ -8,12 +8,14 @@ $(document).ready ->
         redirectUrl = appRouter.generateUrl 'showDashboard'
         displayMenu(redirectUrl)
       else
-        xhrFifo.push(xhr)
+        xhrFifo.push({xhr:xhr, method: settings.type.toUpperCase()})
         context = settings.context
         displayLoader(context.button) if context != undefined && context.button != undefined
     abortXhr: ->
       for i of xhrFifo
-        xhrFifo[i].abort()
+        if xhrFifo[i].method != 'POST'
```

Comment:
If my memory is good, we said that the only type or cancellable requests where the 'GET' requests